src/frontend/src/pages/MainPage/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { useContext, useEffect } from "react";
+import { COLUMN_DIV_STYLE, NAV_DISPLAY_STYLE } from "../../constants";
+
+  import { Download, Upload, Plus, Home, ExternalLink } from "lucide-react";
+  import { TabsContext } from "../../contexts/tabsContext";
+  import { Button } from "../../components/ui/button";
+  import { Link, useNavigate } from "react-router-dom";
+  import { CardComponent } from "../../components/cardComponent";
+  export default function HomePage() {
+    const { flows, setTabId, downloadFlows, uploadFlows, addFlow, removeFlow } =
+    useContext(TabsContext);
+    useEffect(() => {
+      setTabId("");
+    }, []);
+    const navigate = useNavigate();
+  return (
+    <div className={`${ COLUMN_DIV_STYLE }`} >
```

Comment:
Is the string interpolation required here? Wouldn't just `{COLUMN_DIV_STYLE}` work?

Reply:
It would, but i think that this way we can add more styles easier in the future. I can remove it if you want.